### PR TITLE
page detail

### PR DESCRIPTION
**Pull request by luyentap**

### CODE REVIEW

**Comment by luyentap** [comment]
https://edu-redmine.sun-asterisk.vn/issues/8247

**Comment by luyentap** [comment]
ready

File: src/containers/detail/MoreDetail.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    const {id, products} = this.props;
+    let product = products[id-1];
+    return (
+      <div className="more-detail">
+        <div className="description-product">
+          <div className="description-head"><p className="description-head--seleted">Đặc điểm nổi bật</p>
+            <p>Thông tin sản phẩm</p>
+            <p>Đánh giá</p></div>
+          <div className="description-body">
+            <div className="item">
```
Comment (chien-pv) [inline src/containers/detail/MoreDetail.js]: tách ra item là 1 component nhé

**Comment by chien-pv** [comment]
commented

**Comment by luyentap** [comment]
ready

**Pull request merged**